eda/setup.py

```diff
@@ -29,6 +29,7 @@
     'phik>=0.12.2,<0.13',
     'seaborn>=0.12.0,<0.13',
     'ipywidgets>=7.7.1,<9.0',  # min versions guidance: 7.7.1 collab/kaggle
+    'shap>=0.41,<0.42',
```

Comment:
@gradientsky FYI, fastshap may be of interest to test out and compare performance as suggested here: https://github.com/autogluon/autogluon/issues/2222#issuecomment-1455126633

fastshap claims to be much faster than shap:

https://raw.githubusercontent.com/AnotherSamWilson/fastshap/master/benchmarks/iris_benchmark_time.png

Reply:
It doesn't have row explanations and not compatible with shap visualization tools. I tried to plug it, but the values it generates are not matching `shap` outputs.

Reply:
Did a code rework. The code now split into backend analysis and rendering parts. Analysis supports both `shap` and `fastshap` libraries (two different backends). Using `shap` as a default one for `auto` functionality because it is faster.
Visualizations also split into separate primitives; compatible with both of the backends.